content/en/docs/guides/wait-for-ready.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Wait For Ready
+description: >-
+  Explains how and why to configure the wait for ready feature.
+---
+
+### Overview
+
+This is a feature which can be used on a stub which will cause the rpcs to wait
+(until an optional deadline is exceeded) for the server to become available
+before sending the request.
+
+This allows for robust batch workflows as transient server problems (including
+startup) won't cause job failures.
```

Comment:
Should we say "job failures"  or "batch failures"?

Reply:
Removed "job" so it is now just "failures" which is more accurate; the call failed, not necessarily the whole process.